### ISSUE TITLE
Sequence editor

### ISSUE THREAD

**Issue by greenpeas** [dbeaver/dbeaver#2013]
DB 4.1.2
Postgresql 9.6

![image](https://user-images.githubusercontent.com/3856344/29205700-26fda5c4-7e87-11e7-89e3-60449dda923a.png)

**Comment by greenpeas** [comment]
I think it would be great. ))

**Comment by aledmb** [comment]
yeah, this would be a very handy feature. ;)

**Comment by BorjaMinguela** [comment]
Problem persists with postgre 10.5 and dbeaver 5.3.2
I noticed that when chaning directly from th data it tries to execute the following:

```
UPDATE public.test_id_seq
SET "last_value"=42
WHERE "last_value"=64 AND log_cnt=0 AND is_called=false;
```

instead of:

```
ALTER sequence public.test_id_seq
RESTART WITH 42;
```

**Comment by wolandtel** [comment]
Yep. We need the sequence editor.

**Comment by uslss** [comment]
obsolete

**Comment by vhermecz** [comment]
Beg your pardon, why is this actually closed?
This is a feature request not a bug.
This feature is still missing from the product AFAIK.
NOTE: Running 7.0.0, I cannot find this feat, checked release notes of 7.0.1, 7.0.2, those do not mention a sequnce editor either.
I ended up on this page, because I was searching for a way to edit a sequence, without writing SQL.
Thx.

**Comment by marcoregueira** [comment]
It would be great to have this feature. I can't see why it could be "obsolete".

**Comment by mohr023** [comment]
@uslss can you clarify on why this was closed? Is there a commit with this feature that didn't reference this issue?

**Comment by hmcclungiii** [comment]
I just ran into this issue as well.  It occurred as a result of copy/pasting rows from one db to another.  My "hacky" solution was to duplicate rows (which it would not allow me to persist obviously, because of the sequence being behind) causing the sequence value to increment until it finally "caught up"

There is no reason for this to be closed.  Looks to me like it's a feature needed.

**Comment by rajdeep-biswas** [comment]
> @uslss can you clarify on why this was closed? Is there a commit with this feature that didn't reference this issue?

it's because they're useless

**Comment by uslss** [comment]
Reopened.
The issue was closed by mistake.

**Comment by ShadelessFox** [comment]
I believe this issue was resolved by #11177. If not, I will re-open this one.
